README.md

```diff
@@ -61,5 +61,38 @@ this will be interpolated to:
 
 If run-in-terminal can't determine launcher or file_path (file not saved and have no name) it will do start-terminal-here.
 
+## A real life example
+MacOS ver. 10.12.1, runing Atom ver. 1.11.2, using run-in-terminal ver. 0.6.0. The tested program is called test.py and in order to run the script successfully, one first change the python script into excutable:
+
+chmod +x test.py
+
+Then, one has to add the shebang into the first line of the python script:
+
+ \#!/usr/bin/env python
+
+Then one has to configure in the Settings:
+
+1. In 'List of launchers by extension': 
+
+    .py {file_path}
```

Comment:
In my case, nothing can be put between .py and {file_path}, otherwise it won't work.